libr/anal/p/anal_ppc_cs.c

```diff
@@ -18,7 +18,11 @@ struct Getarg {
 	int bits;
 };
 
+#ifdef _MSC_VER
+#define esilprintf(op, fmt, ...) r_strbuf_appendf (&op->esil, fmt, ##__VA_ARGS__)
```

Comment:
i can confirm that __VA_ARGS__ works on gcc/clang. so pls remove all those _MSC_VER ifdefs and just use the portable format